<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditReceive">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditSend"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="buttonSelectFile">
     <property name="text">
      <string>SelectFile</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="buttonSendFile">
     <property name="text">
      <string>SendFile</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="buttonSend">
     <property name="text">
      <string>sendMsg</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="buttonClose">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
